### PR TITLE
Remove Hanami::Utils::Attributes as no longer used

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

**Comment by davydovanton** [comment]
@jodosha where we used this class before?

**Comment by AlfonsoUceda** [comment]
@davydovanton If I remember correctly, in validations

File: test/class_attribute_test.rb [hunk 1]
```diff
@@ -1,152 +0,0 @@
-require 'test_helper'
-require 'hanami/utils/class_attribute'
-
-describe Hanami::Utils::ClassAttribute do
```
Comment (davydovanton) [inline test/class_attribute_test.rb]: I think you need to delete `Attributes` tests too :)

Comment (jodosha) [inline test/class_attribute_test.rb]: OMG, this was the wrong file to remove 😱

**Pull request merged**